staging/kos/pkg/controllers/connectionagent/network_interface.go

```diff
@@ -222,14 +234,40 @@ func (ca *ConnectionAgent) createRemoteNetworkInterface(att *netv1a1.NetworkAtta
 		return nil, err
 	}
 
-	ca.lastClientWriteToRemoteIfcHistograms.
-		WithLabelValues(lastClientWrite).
-		Observe(tAfter.Sub(lastClientWriteTime).Seconds())
-	if lastClientWrite == naLastClientWrite || lastClientWrite == subnetLastClientWrite {
+	lastClientWrName := att.LastClientWrite.Name
+	lastClientWrTime := att.LastClientWrite.Time.Time
+	lastCAStartLabelValue := ""
+	remoteIfcDelayDueToDowntimeSecs := float64(0)
+
+	if lastClientWrTime.Before(vnRelevanceTime) {
+		lastClientWrName = vnRelevanceTrigger + vnRelevanceTriggerLabelSuffix
+		lastClientWrTime = vnRelevanceTime
+
+		if att.LastControllerStart.Controller == netv1a1.LCAControllerStart && ca.startTime.Before(att.LastControllerStart.ControllerTime.Time) {
+			lastCAStartLabelValue = localCAStartLabelValue
+			remoteIfcDelayDueToDowntimeSecs = att.LastControllerStart.ControllerTime.Sub(lastClientWrTime).Seconds()
```

Comment:
Note that we are guaranteed that
```
remoteIfcDelayDueToDowntimeSecs < att.LastControllerStart.ControllerTime.Sub(att.LastClientWrite.Time.Time)
```
and this larger delay is already recorded in a histogram.